### ISSUE TITLE
Add the ability to create/manage custom team roles via Terrraform

### ISSUE THREAD

**Issue by alexandersimpson17** [opsgenie/terraform-provider-opsgenie#346]
Hi!

I would like to control teams and team members via code - but let users control their team settings/schedules via the UI.

This can be done currently - however - anyone who is a team admin can also manually add users via the UI (which we do not want).

This could be fixed by not using the team admin role - but instead creating a custom team role that has all permissions of a team admin - except member management. 

However, creating a custom team role - does not seem to be supported currently by the TF provider?
Can that be added?
Thanks!

**Comment by ncrocfer** [comment]
+1 for this :)

**Comment by slushysnowman** [comment]
Yeah I was just looking for a way to create a custom team role doing exasctly what you described and also ran into this. This would be really handy to have in place
